GDM.HW3.ConditionalsMethods/Program.cs

```diff
@@ -0,0 +1,315 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GDM.HW3.ConditionalsMethods
+{
+    class Program
+    {
+        //Task 2
+        static double Factorial(double fac)
+        {
+            if (fac == 0)
+            {
+                return 1;
+            }
+            else
+            {
+                return fac * Factorial(fac - 1);
+            }
+        }
+
+        //Task 3
+        static int Fibonachi(int n)
+        {
+            return n > 1 ? Fibonachi(n - 1) + Fibonachi(n - 2) : n;
+        }
+
+        //Task 4
+        static int DaysNumberInMonth(int month, int year)
+        {
+            if (month == 2 && year % 4 == 0 && year % 100 != 0 || month == 2 && year % 100 == 0)
+            {
+                return 29;
+            }
+            if (month == 2)
+            {
+                return 28;
+            }
+            if (month % 2 == 1 && month < 7 || month > 8 && month % 2 == 1)
+            {
+                return 30;
+            }
+            else
+            {
+                return 31;
+            }
+        }
+
+        //Task 5
+        static int[] ReccursiveQuickArraySort(int[] array, int firstArrayIndex, int lastArrayIndex)
```

Comment:
Your algorithm is faster in 5-10 times than my example in github :) GREAT JOB:)

Reply:
wow)